documentation/src/main/asciidoc/userguide/appendices/Configurations.adoc

```diff
@@ -610,12 +610,12 @@ The default value of this setting is determined by the value for `hibernate.gene
 [[configurations-cache]]
 === Cache Properties
 
-`*hibernate.cache.region.factory_class*` (e.g. `org.hibernate.cache.infinispan.InfinispanRegionFactory`)::
+`*hibernate.cache.region.factory_class*` (e.g. `org.hibernate.cache.ehcache.internal.EhcacheRegionFactory`)::
```

Comment:
Hi Vlad,

A better example would be `jcache`. The Ehcache region factory is only supporting Ehcache 2 so it's deprecated and we'd better use the short names rather than the internal class names.